src/Illuminate/Support/Str.php

```diff
@@ -340,6 +340,36 @@ public static function limit($value, $limit = 100, $end = '...')
         return rtrim(mb_strimwidth($value, 0, $limit, '', 'UTF-8')).$end;
     }
 
+    /**
+     * Limit the number of characters in a string on the last space before the limit..
+     *
+     * @param  string  $value
+     * @param  int  $limit
+     * @param  string  $end
+     * @param  string  $spaceChar
+     * @return string
+     */
+    public static function limitOnSpace($value, $limit = 100, $end = '...', $spaceChar = ' ')
```

Comment:
You name this function "limitOnSpace", so what is the meaning of $spaceChar? When someone passes "." character, it has to be "limitOnDot", not "limitOnSpace" anymore. You can consider "limitByChar" or something similar.

Reply:
Sure, good point.
Did it for space, and forgot to rename it after adding the extra param.